ansible/playbooks/AdoptOpenJDK_AIX_Playbook/roles/crontab/tasks/main.yml

```diff
@@ -0,0 +1,14 @@
+---
+###########
+# crontab #
+###########
+
+- name: Add cron job to remove stale files from /tmp (owned by jenkins)
+  cron: name="Remove stale files from /tmp (jenkins)" weekday="*" minute="10" hour="0" user=root state=present
+        job="/usr/bin/find /tmp -user jenkins ! -type d -mtime +1 | xargs rm -f"
+  tags: crontab
+
+- name: Add cron job to remove stale directories from /tmp (owned by jenkins)
+  cron: name="Remove stale directories from /tmp (jenkins)" weekday="*" minute="10" hour="2" user=root state=present
+        job="/usr/bin/find /tmp -user jenkins -type d -mtime +1 | sort -r | xargs rm -f"
```

Comment:
Perhaps use a larger value such as `-mtime 4` so that we don't "lose" things that might be useful for debugging problems on the machine that happen, say, over a weekend when no-one will be actively looking at problems. I don't think 4 days of "stuff" accumulated in /tmp will be enough to cause us a problem.

Reply:
Can always make it smaller if needed.
And I corrected a typo! - needs rmdir!